curiouslearning/FeedTheMonsterJS
# fix: CR-151- Bangla word level font issue fixed

# Changes
- ftm_bangla.json
- build folder changes

# How to test
- Check word levels in bangla language.

Ref: [CR-151](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-151)


[CR-151]: https://curiouslearning.atlassian.net/browse/CR-151?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
